content/guides/content-management/06-preview.md

```diff
@@ -20,3 +20,42 @@ To add a comment to a document, open preview and click the "toggle comments" but
 Now place the marker in the document by single clicking the desired place. Write your comment in the text field and click submit. That's it. Existing comments (from all users) will appear in the comment list along with markers in the document (only in preview mode).
 
 ![preview_comment](../img/preview_comment.png)
+
+## Shapes
+
+**Redaction**
+
+Sometimes when you are working with sensitive information in documents you don’t want to share them with everyone, only users having the appropriate privileges. What if you would like to share the document except the sensitive information? This is the case when you need to apply redaction.
+With this feature you can hide parts of your doc with a black rectangles. Opening a document without redaction is only possible if the user has permission for it, otherwise the page will be full with black squares.
```

Comment:
```suggestion
With this feature, you can hide parts of your doc with black rectangles. Opening a document without redaction is only possible if the user has permission for it, otherwise the page will be full with these black shapes.
```